c2corg_ui/static/partials/cards/outings.html

```diff
@@ -45,7 +45,7 @@
       </div>
     </div>
 
-    <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
+    <div class="quality {{'icon'+ doc['quality']}}" ng-if="::doc['quality']" 
```

Comment:
Angular is completely new for me ... I have reproduce what is use for other icon (ng class is not use except for quality) Ok so I move 'icon' to ng-class something like that : ng-class ="'icon'+::doc['quality']" 
I'm not sure of the syntax ...